c7n/query.py

```diff
@@ -460,8 +460,8 @@ class QueryResourceManager(ResourceManager, metaclass=QueryMeta):
 
     source_mapping = sources
 
-    def __init__(self, data, options):
```

Comment:
Seems reasonable to me at a glance - looks like we have `QueryResourceManager` classes in all provider modules though, probably need to update those too or we'll drop consistency in one spot for consistency in another...

I noticed we also have `data, options` args [for `Policy`](https://github.com/cloud-custodian/cloud-custodian/blob/45956b6e38d83e1b7c681dce71d583ef74da032b/c7n/policy.py#L1130) but _that_ seems like it should stay as-is.